README.rst

```diff
@@ -31,58 +39,48 @@ or::
    > stratis --version
 
 To run without installing, check out the source, change to the top directory
-and enter::
+and set ``PYTHONPATH`` environment variable to include library
+dependencies. For example (if using bash shell)::
 
-   > ./bin/stratis --help
+   > export PYTHONPATH="src:../stratisd-client-dbus/src:../into-dbus-python/src:../dbus-signature-pyparsing/src"
+   > python3 bin/stratis --help
 
-making sure that your PYTHONPATH environment variable is set to the CLI src
-directory.
-
-If you are not running the Stratis dbus service most actions will be
-unavailable, but all help menus should work properly.
+Since ``stratis`` uses stratisd's API, most operations will fail unless
+you also running the `Stratis daemon <https://github.com/stratis-storage/stratisd>`_.
 
 You may have to explicitly invoke the Python 3 interpreter if Python 3 is
-not your default Python implementation.
+not your default Python implementation, as shown above.
 
 Testing
 -------
-There are some unit and integration tests in the tests directory.
-
-These can be run by setting your PYTHONPATH environment variable to the CLI
-src directory, setting the STRATIS environment variable to the absolute path
-of your stratisd executable, and entering::
+There are unit and integration tests in the ``tests`` directory.
 
-   > py.test tests
+Running tests depends on some additional packages: python3-pytest,
+python3-hypothesis.
 
-at the command prompt. Entering::
+These can be run by setting the PYTHONPATH environment variable as
+shown above, and also setting the STRATIS environment variable to the
+absolute or relative path to your stratisd executable. For example
+(again using bash shell)::
 
-   > tox
-
-will cause some tests to be run in the tox virtual environment. Integration
-tests, which requires the CLI to connect with the Stratis daemon, will not
-be run.
-
-Note that stratis-cli is a Python 3 program, so it will be necessary for you
-to use the Python 3 variants of pytest and tox that are available on your
-system.
+  > STRATISD=../stratisd/target/debug/stratisd make dbus-tests
```

Comment:
The CI runs the stuff that can be run under tox. It's the only stuff that it can run.